### PR TITLE
utils: addition of script utils

### PR DESCRIPTION
**Pull request by blixhavn**
- NEW Adds script util file for creating XML files based off of
  to_marc21 conversion files.
- NEW Adds script util file for adding order to existing
  conversion files.

Signed-off-by: Øystein Blixhavn oystein@blixhavn.no

### CODE REVIEW

File: .checkignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# This file uses `exec` as it parses a python file.
```
Comment (jirikuncar) [inline .checkignore:1]: ?

Comment (blixhavn) [inline .checkignore:1]: Tried to make it pass QuantifiedCode

**Comment by blixhavn** [comment]
@jirikuncar What's your stance on the `exec` call in this PR? It breaks QuantifiedCode, but I'm hesitant to rewrite my script to avoid it. As this is just a utility script, I don't see the imminent security risk.

**Comment by tiborsimko** [comment]
`scripts/utils_create_xml.py` is too generic a name for my taste.  DoJSON is meant to be doing lots of things, `marc21` is just one of `contrib` things. We may have later `ead`, `pcdm`, and whatnot other parsers.  Hence the helper scripts should be at the very least named in an appropriate manner, e.g. `marc21_add_order`, `marc21_filter_fields`, and whatnot. Also, they may be perhaps better living under the `contrib` directory, since they are not strictly speaking DoJSON-wide applicable, but rather contrib/marc specific...

**Comment by jirikuncar** [comment]
@tiborsimko it can live under `/scripts/contrib/marc21/...` but definitely not inside `/dojson` Python package.

**Comment by jirikuncar** [comment]
IMHO these utility functions can live outside the repo. If there is a wider interest we can re-open this PR.

**Pull request closed without merging**